contracts/ceros/ClisToken.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.10;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "./NonTransferableERC20.sol";
+
+
+contract ClisToken is OwnableUpgradeable, NonTransferableERC20 {
+    /**
+     * Variables
+     */
+    mapping(address => bool) public _minters;
+
+    /**
+     * Events
+     */
+    event MinterModified(address minter, bool isAdd);
+
+    /**
+     * Modifiers
+     */
+    modifier onlyMinter() {
+        require(_minters[msg.sender], "Minter: not allowed");
+        _;
+    }
+
+    function initialize(string memory name, string memory symbol) external initializer {
```

Comment:
Maybe no need to specify `memory` as they are `memory` by default?

Reply:
it's fine to keep it in a more explicit form?